content/en/docs/Configuration/namespace-management.md

```diff
@@ -5,188 +5,201 @@ description: "Configuring the namespaces accessible and visible to Kiali."
 
 ## Introduction
 
-The default Kiali installation (as mentioned in the [Installation guide]({{< ref
-"/docs/installation/installation-guide" >}})) gives Kiali access to all
-namespaces available in the cluster.
+The default Kiali [installation]({{< ref "/docs/installation/installation-guide" >}}) gives Kiali access to all namespaces available in the cluster and will allow all namespaces to be visible.
 
-It is possible to restrict Kiali to a set of desired namespaces by providing a list
-of the ones you want, excluding the ones you don't want. You can also filter namespaces by using [Istio's discovery selectors](https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig). You can use a combination of these options together.
+It is possible to restrict Kiali so that it can only access a specific set of namespaces by providing [discovery selectors](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#resources-that-support-set-based-requirements) that match those namespaces. Note that Kiali will not use [Istio's discovery selectors](https://istio.io/latest/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig); if Istio has been configured with its own discovery selectors, you will likely want to configure Kiali with the same list of discovery selectors.
+
+{{% alert color="info" %}}
+This documentation makes a distinction between a namespace that is _accessible_ verses _visible_. Kiali can be given access to all namespaces in the cluster, or it can be given restricted permissions so that only certain namespaces are accessible to it. Of the accessible namespaces that Kiali is permitted to see, it can make some or all of them visible to end users.
+{{% /alert %}}
 
 {{% alert color="warning" %}}
-As of Kiali 1.67, the following settings are deprecated:
+As of Kiali 1.89, the following settings are no longer supported:
```

Comment:
2.0?

Reply:
that's weird.. I fixed that yesterday.. lemme see where that went :)

Reply:
ah! its in my other branch where I'm getting ready to add back the istio discovery selectors https://github.com/jmazzitelli/kiali.io/commit/ff3ea7bea8886e4dc7bc539ea1ae6674f69735d8#diff-18799846877673dfb4a2b103af33b69fece2ba435d3c326fa4c55ac7bc480920R17

Let me put that in this PR

Reply:
done.